ui/cypress.config.ts

```diff
@@ -44,7 +44,7 @@ export default defineConfig({
             return require('./cypress/plugins/index.ts')(on, config);
         },
         specPattern: 'cypress/tests/**/*.{js,jsx,ts,tsx}',
-        baseUrl: 'http://localhost:80',
+        baseUrl: 'http://localhost:8082',
```

Comment:
I assume this was committed by accident, right?

Reply:
Ah yes, I accidentally did "git commit -am" instead of only committing one file. I have reverted all the irrelevant changes. Thank you for catching it!